tests/ui-fulldeps/fluent-messages/test.stderr

```diff
@@ -93,6 +93,14 @@ LL |         test_crate => "./missing-crate-name.ftl",
    |
    = help: replace any '-'s with '_'s
 
-error: aborting due to 10 previous errors
+error: referenced message `message` does not exist
+  --> $DIR/test.rs:104:20
+   |
+LL |         missing => "./missing-message-ref.ftl"
```

Comment:
This span is...... strange I would say. It's probably not possible to render this with a span pointing to the error in the ftl file, but is it possible for us to mention what fluent slug it comes from (i.e. in this case `missing_message_ref`)?